src/lib/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@react-native-common-modules",
+  "version": "0.0.1",
+  "main": "src/index.ts",
```

Comment:
I think we should also specify which files are included into the package using `files` array - [example](https://github.com/RonasIT/rtkq-entity-api/blob/main/package.json#L28C3-L30C5)